src/traits.rs

```diff
@@ -753,6 +753,16 @@ pub trait SystemExt: Sized + Debug + Default {
     /// ```
     fn get_free_memory(&self) -> u64;
 
+    /// Returns the amount of available RAM in kB.
```

Comment:
Just realized but I think it should `KB` and not `kB` (since we divide by 1000 and not 1024). I'll need to update the others as well.

Reply:
From some cursory searching, I think `kB` is the least ambiguous term to denote 1000 bytes (though some people still use it to mean 1024 bytes).

The IEC standard is that `kB` is 1000 and `KiB` is 1024 [[1]](
https://physics.nist.gov/cuu/Units/binary.html).

There also appears to be a JEDEC standard where `KB` is 1024 bytes [[2]](https://en.wikipedia.org/wiki/Kibibyte).

Reply:
You're absolutely right! Always lost between the different units... Thanks a lot for the explanation!